runestone/activecode/test/test_activecode.py

```diff
@@ -5,7 +5,7 @@
 from runestone.unittest_base import module_fixture_maker, RunestoneTestCase
 
 setUpModule, tearDownModule = module_fixture_maker(__file__)
-
+print "test_activecode.py"
```

Comment:
The runestone components are Python3 compatible, so print should use parenthesis.  Is this also debug code that should be removed?